package.json

```diff
@@ -26,50 +26,50 @@
     "lint": "eslint --ext .ts,.svelte, ./src/elements",
     "test": "npx playwright test",
     "test-dev": "npx playwright test --debug",
-    "storybook": "start-storybook -p 6006",
-    "storybook-docs": "start-storybook --docs --no-manager-cache",
-    "build-storybook": "build-storybook --docs -o prime"
+    "storybook": "storybook dev -p 6006",
+    "storybook-docs": "storybook dev --docs --no-manager-cache",
+    "build-storybook": "storybook build --docs -o prime"
   },
   "devDependencies": {
-    "@babel/core": "^7.19.6",
-    "@floating-ui/dom": "^1.0.4",
-    "@playwright/test": "^1.28.1",
-    "@storybook/addon-a11y": "^6.5.13",
-    "@storybook/addon-actions": "^6.5.13",
-    "@storybook/addon-docs": "^6.5.13",
-    "@storybook/addon-essentials": "^6.5.13",
-    "@storybook/addon-interactions": "^6.5.13",
-    "@storybook/addon-links": "^6.5.13",
-    "@storybook/builder-vite": "^0.2.5",
-    "@storybook/html": "^6.5.13",
-    "@storybook/testing-library": "^0.0.13",
-    "@sveltejs/vite-plugin-svelte": "^1.1.0",
+    "@floating-ui/dom": "^1.0.12",
+    "@playwright/test": "^1.29.0",
+    "@storybook/addon-a11y": "^7.0.0-beta.13",
+    "@storybook/addon-actions": "^7.0.0-beta.13",
+    "@storybook/addon-docs": "^7.0.0-beta.13",
+    "@storybook/addon-essentials": "^7.0.0-beta.13",
+    "@storybook/addon-interactions": "^7.0.0-beta.13",
+    "@storybook/addon-links": "^7.0.0-beta.13",
+    "@storybook/html": "^7.0.0-beta.13",
+    "@storybook/html-vite": "^7.0.0-beta.13",
+    "@storybook/testing-library": "^0.0.14-next.1",
+    "@sveltejs/vite-plugin-svelte": "^2.0.2",
     "@tsconfig/svelte": "^3.0.0",
-    "@typescript-eslint/eslint-plugin": "^5.42.0",
-    "@typescript-eslint/parser": "^5.42.0",
-    "babel-loader": "^9.1.0",
+    "@typescript-eslint/eslint-plugin": "^5.47.0",
+    "@typescript-eslint/parser": "^5.47.0",
     "classnames": "^2.3.2",
-    "element-internals-polyfill": "^1.1.16",
-    "eslint": "^8.26.0",
+    "element-internals-polyfill": "^1.1.17",
+    "eslint": "^8.30.0",
     "eslint-import-resolver-typescript": "^3.5.2",
     "eslint-plugin-import": "^2.26.0",
     "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-prefer-arrow": "^1.2.3",
-    "eslint-plugin-sonarjs": "^0.16.0",
+    "eslint-plugin-sonarjs": "^0.17.0",
+    "eslint-plugin-storybook": "^0.6.8",
     "eslint-plugin-svelte3": "^4.0.0",
-    "eslint-plugin-tailwindcss": "^3.6.2",
-    "eslint-plugin-unicorn": "^44.0.2",
+    "eslint-plugin-tailwindcss": "^3.7.1",
+    "eslint-plugin-unicorn": "^45.0.2",
     "monaco-editor": "^0.34.1",
-    "postcss": "^8.4.18",
-    "svelte": "^3.52.0",
-    "svelte-check": "^2.9.2",
-    "svelte-loader": "^3.1.3",
-    "svelte-preprocess": "^4.10.7",
-    "tailwindcss": "^3.2.1",
-    "terser": "^5.15.1",
+    "postcss": "^8.4.20",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "storybook": "^7.0.0-beta.13",
```

Comment:
This is only made obvious by the `package-lock.json`, but dependencies have been substantially reduced.